Okta provider - error in type conversion

I'm testing the terraformer and it throws this error for Okta provider:

```shell
2024-08-26T19:26:32.355+0200 [ERROR] plugin.terraform-provider-okta_v4.10.0: Response contains error diagnostic: @module=sdk.proto tf_proto_version=5.4 @caller=github.com/hashicorp/terraform-plugin-go@v0.22.2/tfprotov5/internal/diag/diagnostics.go:58 diagnostic_attribute=""
  diagnostic_detail=
  | An unexpected error was encountered trying to build a value. This is always an error in the provider. Please report the following to the provider developer:
  |
  | Received null value, however the target type cannot handle null values. Use the corresponding `types` package type, a pointer type or a custom type that handles null values.
  |
  | Path:
  | Target Type: okta.FrameworkProviderData
  | Suggested `types` Type: basetypes.ObjectValue
  | Suggested Pointer Type: *okta.FrameworkProviderData
   diagnostic_severity=ERROR diagnostic_summary="Value Conversion Error" tf_provider_addr=okta/okta tf_rpc=Configure tf_req_id=eb11a554-689f-67a7-1838-e99a1b375161 timestamp="2024-08-26T19:26:32.355+0200"
 ```
 
Using Terraform v1.7.4  and the Okta provider v4.10.0

--- a/providers/okta/okta_provider.go
+++ b/providers/okta/okta_provider.go
@@ -20,6 +20,7 @@ import (
 
 	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
 	"github.com/GoogleCloudPlatform/terraformer/terraformutils/providerwrapper"
+	"github.com/zclconf/go-cty/cty"
 )
 
 type OktaProvider struct { //nolint
@@ -127,4 +128,13 @@ func (p *OktaProvider) GetSupportedService() map[string]terraformutils.ServiceGe
 		"okta_user_schema":               &UserSchemaPropertyGenerator{},
 		"okta_app_user_schema":           &AppUserSchemaPropertyGenerator{},
 	}
+}
+
+// GetConfig returns the provider configuration for Okta
+func (p *OktaProvider) GetConfig() cty.Value {
+	return cty.ObjectVal(map[string]cty.Value{
+		"base_url":  cty.StringVal(p.baseURL),
+		"api_token": cty.StringVal(p.apiToken),
+		"org_name":  cty.StringVal(p.orgName),
+	})
 }

